pkgs/tools/text/ansifilter/default.nix

```diff
@@ -0,0 +1,22 @@
+{ stdenv, pkgs }:
+let version = "1.15";
+    pkgsha = "65dc20cc1a03d4feba990f830186404c90462d599e5f4b37610d4d822d67aec4";
+in with pkgs; stdenv.mkDerivation {
+  name = "ansifilter";
+  buildInputs = [
+    stdenv gcc gnumake
+    pkgconfig boost lua
+  ];
+  src = fetchurl {
+    url = "http://www.andre-simon.de/zip/ansifilter-${version}.tar.bz2";
+    sha256 = pkgsha;
+  };
+  meta = {
```

Comment:
The `meta` attribute needs to be fleshed out; at least provide license and a proper description.